src/pages/auth/chat/room/ExitBtn.tsx

```diff
@@ -21,7 +21,7 @@ export default function ExitBtn(props: { room: number }) {
     return () => {
       socket.off("exitChatRoomResult", listner);
```

Comment:
```suggestion
      socket.off("exitChatRoomResult", listener);
```

이거 전에 발견 못했는데, 오타 있어요..!ㅎㅎ